DependencyInjection/DoctrineMigrationsExtension.php

```diff
@@ -25,8 +34,95 @@ public function load(array $configs, ContainerBuilder $container) : void
 
         $config = $this->processConfiguration($configuration, $configs);
 
-        foreach ($config as $key => $value) {
-            $container->setParameter($this->getAlias() . '.' . $key, $value);
+        $configurationDefinition = new Definition(MigrationsConfiguration::class);
+        $container->setDefinition('doctrine.migrations.configuration', $configurationDefinition);
+
+        if (! empty($config['name'])) {
+            $configurationDefinition->addMethodCall('setName', [$config['name']]);
+        }
+        if (! empty($config['migrations_paths'])) {
+            foreach ($config['migrations_paths'] as $ns => $path) {
+                $configurationDefinition->addMethodCall('addMigrationsDirectory', [$ns, $path]);
+            }
+        }
+        if (isset($config['all_or_nothing'])) {
```

Comment:
This condition looks wrong, I think it should be
```suggestion
        if ($config['all_or_nothing']) {
```